test/jasmine/tests/pie_test.js

```diff
@@ -522,33 +478,44 @@ describe('Test event property of interactions on a pie plot:', function() {
         it('should contain the correct fields', function() {
             mouseEvent('mouseover', pointPos[0], pointPos[1]);
 
-            var point0 = futureData.points[0],
-                evt = futureData.event;
-            expect(point0.originalEvent).toEqual(evt, 'points');
-            expect(evt.clientX).toEqual(pointPos[0], 'event.clientX');
-            expect(evt.clientY).toEqual(pointPos[1], 'event.clientY');
+            checkEventData(futureData);
+        });
+
+        it('should not emit a hover if you\'re dragging', function() {
+            gd._dragging = true;
```

Comment:
It occurred to me when fixing this bit of logic, it would perhaps be better to check `event.buttons` instead? Seems both cleaner and more powerful, for example it would prevent hover if you drag over a plot from somewhere else on the page. So in principle perhaps *all* our hover handlers should contain this check?

Reply:
that will need a bunch more testing and ideally rolling out to all subplots at once, so I'll defer it -> #2120